### PR TITLE
Result view issue fix

### PR DESCRIPTION
**Pull request by pkumarmshetty**
The QR file upload and scan upload issue has been fixed.

### CODE REVIEW

**Comment by swatigoel** [comment]
@pkumarmshetty This PR is raised against master branch. Shouldn't we raise it against develop branch so that it goes through QA cycle.

File: helm/inji-verify/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: injiverify
 description: A Helm chart for MOSIP injiverify module
 type: application
-version: 0.0.1-develop
+version: 0.8.1
```
Comment (swatigoel) [inline helm/inji-verify/Chart.yaml:5]: chart version 0.9.0 is released 2 weeks back. So let's have it as 0.9.1

File: helm/inji-verify/install.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@ if [ $# -ge 1 ] ; then
 fi
 
 NS=injiverify
-CHART_VERSION=0.0.1-develop
+CHART_VERSION=0.8.1
```
Comment (swatigoel) [inline helm/inji-verify/install.sh:10]: chart version 0.9.0 is released 2 weeks back. So let's have it as 0.9.1

File: helm/inji-verify/install.sh [hunk 3]
```diff
@@ -31,13 +31,12 @@ fi
 
 echo "MOSIP_INJIVERIFY_HOST is not present in configmap/global of configserver"
     # Add injiverify host to global
-    kubectl patch configmap global -n config-server --type merge -p "{\"data\": {\"mosip-injiverify-host\": \"$MOSIP_INIJIVERIFY_HOST\"}}"
+    kubectl patch configmap global -n config-server --type merge -p "{\"data\": {\"mosip-injiverify-host\": \"$MOSIP_INJIVERIFY_HOST\"}}"
+    kubectl patch configmap global -n default --type merge -p "{\"data\": {\"mosip-injiverify-host\": \"$MOSIP_INJIVERIFY_HOST\"}}"
```
Comment (swatigoel) [inline helm/inji-verify/install.sh:35]: Why do we need this configuration in global namespace?

File: helm/inji-verify/install.sh [hunk 4]
```diff
@@ -31,13 +31,12 @@ fi
...
     # Add the host
     kubectl set env deployment/config-server SPRING_CLOUD_CONFIG_SERVER_OVERRIDES_MOSIP_ESIGNET_INJIVERIFY_HOST=$MOSIP_INJIVERIFY_HOST -n config-server
     # Restart the configserver deployment
-    kubectl -n config-server get deploy -o name | xargs -n1 -t kubectl -n config-server rollout restart 
-
-sleep 400s
+    kubectl -n config-server get deploy -o name | xargs -n1 -t kubectl -n config-server rollout status
```
Comment (swatigoel) [inline helm/inji-verify/install.sh:39]: Can you please let us know the reason for replacing restart by status?

File: helm/inji-verify/values.yaml [hunk 5]
```diff
@@ -416,6 +415,11 @@ metrics:
 
 ## The url below is to access esignet API services.
 ## CAUTION: Domain name to access injiweb UI must be secure/internal/over vpn.
+## Slash is important!
+inji_verify:
+  port: '3004'
+  host: 'mimoto.mimoto'
```
Comment (swatigoel) [inline helm/inji-verify/values.yaml:421]: why mimoto host is needed here?

File: helm/inji-verify/values.yaml [hunk 6]
```diff
@@ -416,6 +415,11 @@ metrics:
...
+esignet_redirect_url: 'esignet.sandbox.xyz.net'
```
Comment (swatigoel) [inline helm/inji-verify/values.yaml:422]: why do we need esignet redirect uri?

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: inji-verify/src/utils/qr-utils.js [hunk 7]
```diff
@@ -1,11 +1,16 @@
...
 
 export const scanFilesForQr = async (selectedFile) => {
     let scanResult = { data: null, error: null };
     try {
-        scanResult.data = await scanFile(selectedFile);
+        const data = await QrScanner.scanImage(selectedFile)
+        .then(result => result)
+        .catch(error => console.log(error || 'No QR code found.'));
+        scanResult.data = data;
+        console.log(scanResult.data);
```
Comment (swatigoel) [inline inji-verify/src/utils/qr-utils.js:13]: Removing console log for scanned result

File: inji-verify/src/components/Home/VerificationSection/QrScanner.tsx [hunk 8]
```diff
@@ -42,10 +42,10 @@ function QrScanner({setActiveStep, setQrData}: {
     return (
         <div ref={scannerRef}>
             <Scanner
-                onResult={(text, result) => {
-                    console.log(text, result);
+                onScan={(text, result) => { 
+                    console.log(text,text);
```
Comment (swatigoel) [inline inji-verify/src/components/Home/VerificationSection/QrScanner.tsx:46]: remove console log

File: inji-verify/src/utils/qr-utils.js [hunk 9]
```diff
@@ -1,11 +1,16 @@
 import { scanFile } from "@openhealthnz-credentials/pdf-image-qr-scanner";
```
Comment (swatigoel) [inline inji-verify/src/utils/qr-utils.js:1]: This is unused now. We can remove `@openhealthnz-credentials/pdf-image-qr-scanner` as started using `qr-scanner`

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: helm/inji-verify/install.sh (hunk 3 above)
Comment (ckm007) [reply helm/inji-verify/install.sh:35]: Better it is updatd in global also. Looks good only

**Comment by ckm007** [comment]
Can see there are conflicts as well. Please resolve the same when raising PR for develop.

**Comment by swatigoel** [comment]
DCO check is also failing for last 2 commits

**Comment by pkumarmshetty** [comment]
closeing this PR created another PR(Qrscanner issue fix ) pointing to develop branch

**Pull request closed without merging**